omero/sysadmins/unix/server-installation.txt

```diff
@@ -248,22 +248,23 @@ Ice
 .. note::
 
     OMERO |version| supports 3.5 and 3.4 on UNIX and UNIX-like
-    platforms. You must install the correct version of OMERO.server
-    (see :downloads:`Downloads <>`).
+    platforms; support for 3.6 is currently upcoming. You must install
+    the correct version of OMERO.server (see :downloads:`Downloads
+    <>`).
 
 If possible, install one of the following packages:
 
-+-----------+---------------------------------+
-| System    | Package                         |
-+===========+=================================+
-| BSD Ports | devel/ice                       |
-+-----------+---------------------------------+
-| Debian    | zeroc-ice                       |
-+-----------+---------------------------------+
-| Homebrew  | ice (with --with-python option) |
-+-----------+---------------------------------+
-| RedHat    | N/A (install ZeroC RPMs)        |
-+-----------+---------------------------------+
++-----------+-------------------------------+
+| System    | Package                       |
++===========+===============================+
+| BSD Ports | devel/ice                     |
++-----------+-------------------------------+
+| Debian    | zeroc-ice                     |
++-----------+-------------------------------+
+| Homebrew  | ice (with --with-java option) |
++-----------+-------------------------------+
+| RedHat    | N/A (install ZeroC RPMs)      |
++-----------+-------------------------------+
 
 The Ice version may vary, depending upon the distribution version you
 are using. The Ice versions in currently supported versions of Debian
```

Comment:
`are shown in the :ref:ice-requirements of the :doc:/sysadmins/version-requirements page` (a couple lines below) comes out as `are shown in the Ice of the Version requirements page.`